aptos-move/aptos-vm/src/counters.rs

```diff
@@ -78,30 +82,59 @@ pub static SYSTEM_TRANSACTIONS_EXECUTED: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
+const NUM_BLOCK_TRANSACTIONS_BUCKETS: [f64; 30] = [
```

Comment:
I feel this is too many. May be reduce the number of buckets below 100 txns? And, cap to 30k for now, until we hit 50k? This should get us down to 25 buckets.

Reply:
+1. My recommendation would be to try and limit each one two 20 buckets. More can be used, but it seems like it would require further justification 😄 Otherwise, displaying these things will become a nightmare 😱

Reply:
Reduced the number of buckets now.